mne/viz/_brain/tests/test_brain.py

```diff
@@ -358,12 +357,14 @@ def __init__(self):
     fp = np.array(
         brain._renderer.figure.plotter.renderer.ComputeVisiblePropBounds())
     fp = (fp[1::2] + fp[::2]) * 0.5
-    view_args = dict(azimuth=180., elevation=90., focalpoint='auto')
-    brain.show_view(view=view_args)
-    assert np.allclose(brain._renderer.figure._azimuth, view_args["azimuth"])
-    assert np.allclose(
-        brain._renderer.figure._elevation, view_args["elevation"])
-    assert np.allclose(cam.GetFocalPoint(), fp)
+    azimuth, elevation = 180., 90.
+    for view_args in (dict(azimuth=azimuth, elevation=elevation,
+                           focalpoint='auto'),
+                      dict(view='lateral', hemi='lh')):
+        brain.show_view(**view_args)
+        assert np.allclose(brain._renderer.figure._azimuth, azimuth)
```

Comment:
use `assert_allclose` rather than `assert np.allclose`, the error message is better when it fails and it has better defaults for testing (`atol=0` which makes more sense than `atol=1e-7` in general)